elementary/utils/log.py

```diff
@@ -35,8 +37,21 @@ def get_console_handler():
     return console_handler
 
 
+def get_log_path():
+    file_path = os.getcwd() + "/edr_target/"
+    try:
+        ctx = click.get_current_context()
+        ctx_args = ctx.args
+        target_path_flag = "--target-path"
+        file_path = ctx_args[ctx_args.index(target_path_flag) + 1]
+    finally:
+        os.makedirs(os.path.abspath(file_path), exist_ok=True)
+        return file_path + "/edr.log"
```

Comment:
it's a good practice in python usually to use `os.path.join(file_path, "edr.log")` rather than adding strings 
(handles well edge cases, like if the path ends or doesn't end with "/")

Reply:
Ha good to know, I saw both in the code so just used one of them 👍